SoD_SP_Public/Next Version/SoD_SP.ini

```diff
@@ -31,37 +31,37 @@ SoD_SPLanguage = eng.json
 # Цвет координат курсора мыши на Карте Приключений
 # Die Koordinaten des Mauszeigers während man auf der Abenteuerkarte ist
 # Συντεταγμένες του κέρσορα του ποντικιού όταν βρίσκεστε στον χάρτη περιπέτειας (Adventure Map)
-AdventureCoord = H3Cyan
+AdventureCoord = Regular
 
 # The color of hit points of the creature under the mouse cursor while in combat
 # Цвет здоровья стека существ при наведении курсора мыши во время боя
 # Die Farbe der Lebenspunkte der Kreatur unter dem Mauszeiger während der Kampf
 # Το χρώμα των πόντων ζωής του πλάσματος κάτω από τον κέρσορα του ποντικιού κατά τη διάρκεια της μάχης
-HitPoints = H3Orange
+HitPoints = Regular
```

Comment:
The default ini options should not be modified, just the comments explaining what they do.